src/main/java/frc/robot/robotcontainers/BeetleContainer.java

```diff
@@ -22,19 +23,26 @@ public class BeetleContainer extends RobotContainer {
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public BeetleContainer() {
     // Configure the button bindings
-    configureButtonBindings();
+        configureButtonBindings();
   }
 
+private void configureButtonBindings(){
```

Comment:
This method is better to be changed back to its original place, i.e. after line 42, with comments there.

Reply:
code moved as suggested